connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConnector.java

```diff
@@ -96,6 +100,7 @@ public class MirrorSourceConnector extends SourceConnector {
     private Admin sourceAdminClient;
     private Admin targetAdminClient;
     private Admin offsetSyncsAdminClient;
+    private String useIncrementalAlterConfigs;
```

Comment:
Since the value of this field can be altered on a different thread from the one that checks it, I think we want to add the `volatile` keyword here.